### ISSUE TITLE
English translation

### ISSUE THREAD

**Issue by scriptPilot** [jekip/naive-ui-admin#202]
This UX Framework is awesome. It would be great to have an English-first approach. Today, even the demo is not usable for most people from outside China.

**Comment by jekip** [comment]
Indeed, multilingualism was not planned from the beginning vue ecology  There are relatively mature solutions and I believe it is easy to integrate myself

**Comment by scriptPilot** [comment]
@jekip - Yes, for Vue there is https://vue-i18n.intlify.dev/ available. Would be great of the demo app would support that and the project documentation would be translated to English as well.

**Comment by cuiyiming1998** [comment]
> @jekip - Yes, for Vue there is https://vue-i18n.intlify.dev/ available. Would be great of the demo app would support that and the project documentation would be translated to English as well.

#222
